paymentapi/src/main/java/fi/hel/verkkokauppa/payment/service/PaymentPaytrailService.java

```diff
@@ -179,6 +200,10 @@ private Payment createPayment(PaytrailPaymentContext context, GetPaymentRequestD
         return payment;
     }
 
+    private void addPaytrailPaymentCreateResponseDataToPayment(PaytrailPaymentResponse paymentResponse, Payment payment) {
```

Comment:
This should probably be deleted?

Reply:
Oh yeh, I had this on my local as deleted. forgot to push it :D